src/spaceObjects/planet.h

```diff
@@ -44,7 +44,7 @@ class Planet : public SpaceObject, public Updatable
     string planet_texture;
     string cloud_texture;
     string atmosphere_texture;
-    glm::u8vec4 atmosphere_color;
+    glm::u8vec3 atmosphere_color;
```

Comment:
I think for planets it'd be ok since there's not that many (I would assume) that the 8 bytes it saves wouldn't have a huge impact?

I'm not too worried about it either way, it's really potahto potayto so up to you.

Reply:
no reason to save memory, all the reason to make the code simpler ;-)